rules/library.bzl

```diff
@@ -381,6 +381,31 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
     )
     deps += resource_bundles
 
+    # TODO: remove framework if set
```

Comment:
does this mean when framework is set somewhere, we should remove this code? not clear to me what TODO wants

Reply:
When `framework` text is already included in the modulemap, it should be removed before going through this codepath